flashinfer/fused_moe/capabilities.py

```
@@ -0,0 +1,76 @@
+                )
+            )
+
+    return tuple(
+        sorted(rows, key=lambda row: (row.backend_key, row.quant_variant.name))
+    )
+
+
+def render_moe_activation_matrix(
+    rows: tuple[MoEBackendCapability, ...] | None = None,
+) -> str:
+    """Render the generated backend × quantization × activation Markdown table."""
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)